frontend/apps/reader/modules/readertypography.lua

```diff
@@ -395,6 +396,7 @@ These settings will apply to all books with any hyphenation dictionary.
             return self.hyphenation
         end,
     })
+    table.insert(hyphenation_submenu, self.ui.userhyph:getMenuEntry())
     table.insert(hyphenation_submenu, {
         text = _("Trust soft hyphens"),
```

Comment:
It felt better after "Trust soft hyphens". Why did you feel like moving it above?
<kbd>![image](https://user-images.githubusercontent.com/24273478/120112061-9845a400-c174-11eb-84b3-861d3ce12467.png)</kbd>
There's a separator too much :)

Reply:
The idea was to group both global settings together. But I won't mind them the otherway round (without the separator between).

Reply:
But we realized earlier that "Enable hyphenation" was per-document - and Custom hyphenation is not. So yes, please reorder, without the added separator.